cost/azure/rightsize_compute_instances/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v3.3
+
+- Added ability to filter resources by region
```

Comment:
Is adding new parameters a breaking change and therefore this should be version 4.0?

cc - @bryankaraffa

Reply:
@nia-vf1 Only if they don't have default values, and the default values don't enable it to function as it did before the parameter existed.

In this case, simply not specifying a value for the new parameters will result in the policy functioning identically to the previous version, so the change is not breaking.

Reply:
@XOmniverse - You mean only if they do have default values?